integration

```diff
@@ -1163,5 +1164,6 @@
   "zeronounours/HA-custom-component-energy-meter",
   "zhbjsh/homeassistant-ssh",
   "zigul/HomeAssistant-CEZdistribuce",
-  "ZsBT/hass-w1000-portal"
-]
\ No newline at end of file
+  "ZsBT/hass-w1000-portal",
+  "zweckj/acaia"
```

Comment:
You should not be adding `zweckj/acaia` here.

Reply:
This must have been a leftover from a merge conflict. Ill fix this.